dotcom-rendering/src/components/Carousel.importable.tsx

```diff
@@ -30,8 +30,13 @@ type Props = {
 	url?: string;
 	onwardsSource: OnwardsSource;
 	leftColSize: LeftColSize;
-	activeDotColour: string;
+	activeDotColour?: string;
+	titleColour?: string;
 	titleHighlightColour: string;
+	verticalDividerColour?: string;
+	arrowColour?: string;
+	arrowBackgroundColour?: string;
+	arrowBackgroundHoverColour?: string;
```

Comment:
Rather than taking a whole load of colours as props, I think the pattern in DCR would be to pass an optional props of the container palette (`containerPalette?: DCRContainerPalette`), then have the component call decidePalette or `decideContainerOverrides` itself - this is supposed to help us most easily keep track of where we're using colours around the project.

Reply:
we have mobbed on this and have refactored - it's much neater now, thanks Olly and Charlotte!